src/main/resources/galacticraft.mixins.json

```diff
@@ -32,7 +32,8 @@
     "client.InGameHudMixin",
     "client.PauseMenuScreenMixin",
     "client.PlayerInventoryScreenMixin",
-    "client.WorldRendererMixin"
+    "client.WorldRendererMixin",
+    "client.AlphaWarningTitleScreenMixin"
```

Comment:
Put this in alphabetical order

Reply:
This is what I get for blindly following the MC mod dev Intellij plugin :/